central/notifiers/acscsemail/client_impl.go

```diff
@@ -16,10 +30,45 @@ func ClientSingleton() Client {
 		return client
 	}
 
-	client = &clientImpl{}
+	url := fmt.Sprintf("%s:%s", env.ACSCSEmailURL.Setting(), sendMsgPath)
+
+	client = &clientImpl{
+		loadToken:  satoken.LoadTokenFromFile,
+		url:        url,
+		httpClient: http.DefaultClient,
+	}
+
 	return client
 }
 
-func (s *clientImpl) SendMessage(ctx context.Context, msg AcscsMessage) error {
-	return errors.New("TODO: not yet implemented")
+func (c *clientImpl) SendMessage(ctx context.Context, msg AcscsMessage) error {
+	token, err := c.loadToken()
+	if err != nil {
+		return errors.Wrap(err, "failed to load authorization token")
+	}
+
+	msgBytes, err := json.Marshal(&msg)
+	if err != nil {
+		return errors.Wrap(err, "failed to marshal message")
+	}
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, c.url, bytes.NewReader(msgBytes))
+	if err != nil {
+		return errors.Wrap(err, "failed to build HTTP requests")
+	}
+
+	req.Header.Set("Content-Type", "application/json; charset=UTF-8")
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
+
+	res, err := c.httpClient.Do(req)
```

Comment:
nit:  Probably worth a comment about why we are using http instead of smtp.

Reply:
👍 Added a comment to the ClientSingleton func.